tests/include/test/psa_crypto_helpers.h

```diff
@@ -193,6 +193,14 @@ psa_status_t mbedtls_test_record_status( psa_status_t status,
  */
 psa_key_usage_t mbedtls_test_update_key_usage_flags( psa_key_usage_t usage_flags );
 
+/** Check that no PSA Crypto key slots are in use.
+ *
+ * If any slot is in use, mark the current test as failed.
+ *
+ * \return 0 if the key store is empty, 1 otherwise.
+ */
+int test_fail_if_psa_leaking( int line_no, const char *filename );
```

Comment:
This function now has external linkage, so it needs to have a name starting with `mbedtls`.
```suggestion
int mbedtls_test_fail_if_psa_leaking( int line_no, const char *filename );
```

Reply:
makes sense. Done.